go.mod

```diff
@@ -33,6 +33,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.12.0
 )
 
+require github.com/osrg/gobgp v2.0.0+incompatible
```

Comment:
We should only require `gobgp/v3`. You're probably just missing the `/v3` trailer